processing/base_creator.py

```diff
@@ -314,12 +308,12 @@ def driver(self):
         new_entry_ls = []
         base_attrs = {}
         psql_conn = self.db_conn_pool.get('psql_conn')
-        self.site_list = self.flux_db_handler.get_sites_with_updates()
+        self.site_list = self.new_db_handler.get_sites_with_updates()
         self.embargoed_site_list = self.new_db_handler.get_sites_with_embargo(
             psql_conn, self.embargo_years)
         self.historic_site_list = self.new_db_handler.get_sites_with_updates(
             psql_conn, is_historic=True)
-        self.preBASE_files = self.flux_db_handler.get_BASE_candidates()
+        self.preBASE_files = self.new_db_handler.get_base_candidates()
```

Comment:
method has parameter "state_ids".
Old code does not pass anything thru but I'm not see how that worked. There is no default value for state_ids in the method.

Reply:
Thanks. Looks like you updated the original definition on March 18th, but the base_creator code was not updated somehow.

Reply:
I just pushed a fix, should make use of the newer ProcessStatesHandler class